cmd/root.go

```diff
@@ -108,6 +107,27 @@ func Run(spec *Specification, hal hal.InBand) (*event.EventEmitter, error) {
 		return eventEmitter, errors.Wrap(err, "register")
 	}
 
+	grpcClient, err := NewGrpcClient(certsClient, eventEmitter)
+	if err != nil {
+		log.Error("failed to fetch GRPC certificates", "error", err)
+		return eventEmitter, err
+	}
+	hammer.GrpcClient = grpcClient
+
+	skip, err := hammer.CreateBmcSuperuser()
+	if err != nil {
+		log.Error("failed to update bmc superuser password", "error", err)
+		return eventEmitter, err
+	}
+	if !skip {
```

Comment:
I would suggest to put the logic to disable the existing superuser inside the "CreateBMCSuperuser" func